nav2_bringup/params/nav2_params.yaml

```diff
@@ -211,7 +211,7 @@ local_costmap:
         mark_threshold: 0
         observation_sources: scan
         scan:
-          topic: /scan
+          topic: scan
```

Comment:
For here + the global costmap below, add in the example in a comment so that its clear inline here the behavior and recommendation to use the global `/`

```
   * For example:
   *   * User chosen namespace is `tb4`.
   *   * User chosen topic is `scan`.
   *   * Topic will be remapped to `/tb4/scan` without `local_costmap`.
   *   * Use global costmap `/scan` if you do not wish the node namespace to apply
 ```

(I updated it for task)

Reply:
https://github.com/ros-navigation/navigation2/pull/4715/commits/ac5d861dbf44f0c80bd624c4d55722e120c12c28